Makefile

```diff
@@ -116,6 +116,6 @@ version:
 
 .PHONY: run-dev
 run-dev: build
-	NODE_NAME=localhost ./dist/oci-volume-provisioner \
+	NODE_NAME=localhost OCI_VOLUME_PROVISIONER_REGION=iad ./dist/oci-volume-provisioner \
```

Comment:
Perhaps `OCI_VOLUME_PROVISIONER_REGION ?= iad` at the top level would be better to enable overriding.